shimmingtoolbox/read_nii.py

```diff
@@ -0,0 +1,84 @@
+import json
+import math
+import nibabel as nib
+import os
+
+
+def read_nii(niiPath):
+    """
+
+    Args:
+        niiPath (str): Path to the nifti file
+
+    Returns:
+        image:
+        info:
+        jsonData:
+    """
+
+    info = nib.load(niiPath)
+    image = info.get_fdata()
+
+    # `extractBefore` should get the correct filename in both.nii and.nii.gz cases
+    jsonPath = niiPath.split('.nii')[0] + '.json'
+
+    if os.path.isfile(jsonPath):
+        jsonData = json.load(open(jsonPath))
+    else:
+        jsonData = []
+        print('Warning: No json file has been found')
+
+    # NOTE: nib.load automatically scales the nifti and replaces scl_inter and scl_slope with 'nan'
+    # See the "Data scaling" section in https://nipy.org/nibabel/nifti_images.html
+    if ('Manufacturer' in jsonData) and (jsonData['Manufacturer'] == 'Siemens') \
+            and (check_json_image_type(jsonData) == 'phase'):
+
+        image, info = convert_siemens_phase(image, info)
+
+    return image, info, jsonData
+
+# -----------------------------------------------------------------------------
+# Local functions
+# -----------------------------------------------------------------------------
+
```

Comment:
no more needed in python (if it's in the package, it is no more local and can be called from everywhere)-- if you want to make a function private, add "_" as prefix

Reply:
So the idea here is to create a .py file for each function or can I leave them here but just remove the header ?

Reply:
you can leave them for now, and as the project moves forward we will gather API functions in ways that make sense (e.g. a `io.py` file in the library to be able to do `from shimmingtoolbox.io import dcm_to_nii, read_nii`).